### PR TITLE
Adding Django 3.0 support

### LINKED ISSUE #34: Support Django 3.0
Django 3.0 officially removes the django.utils.six module, causing this library to crash with errors like:

```
    from rest_framework_serializer_extensions.utils import (
  File "....../lib/python3.8/site-packages/rest_framework_serializer_extensions/utils.py", line 6, in <module>
    from django.utils import six
ImportError: cannot import name 'six' from 'django.utils' (....../lib/python3.8/site-packages/django/utils/__init__.py)
```

### PR DESCRIPTION
**Pull request by ChrisBurch**
- drops Python 2 support
- updates tox to newer Python/Django/DRF versions
- updates django-test-plus for Django 3.0 support
- updates docs for newly-supported versions

Resolves #34

### CODE REVIEW

**Comment by evenicoulddoit** [comment]
Looks great from first inspection, thanks! I'm not sure how this drops code coverage :thinking:. I was also confused as there's no report of a Travis build, but on closer inspection, it appears to actually be pre-emptively [merging with master and running a build on that](https://travis-ci.org/evenicoulddoit/django-rest-framework-serializer-extensions/builds/623264035/), which is neat, but there's just no feedback!

Post merging this, I'd suggest that I should bump the package to `2.0.0` as dropping Py2 seems cause for a major release?

Hope to get some more time to look at this later and merge :+1:

**Comment by ChrisBurch** [comment]
Sounds good regarding 2.0.0 ! Thanks for taking a look. Yeah I was curious about the coverage drop, too, unsure how that came to be

File: rest_framework_serializer_extensions/serializers.py [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
 from django.db.models.query import QuerySet
-from django.utils import six
 from rest_framework import serializers
 from rest_framework.fields import empty
 
 from rest_framework_serializer_extensions import (
     fields as custom_fields, utils
 )
 
-
```
Comment (evenicoulddoit) [inline rest_framework_serializer_extensions/serializers.py]: This looks like a flake8 regression?

File: tests/test_fields.py [hunk 2]
```diff
@@ -1,13 +1,5 @@
-from __future__ import absolute_import
-
-# Django 1 vs. 2
-try:
-    from django.core.urlresolvers import reverse
-except ImportError:
-    from django.urls import reverse
-
+from django.urls import reverse
```
Comment (evenicoulddoit) [inline tests/test_fields.py]: Alphabetisation is off here too - would you mind?

File: rest_framework_serializer_extensions/serializers.py (hunk 1 above)
Comment (ChrisBurch) [reply rest_framework_serializer_extensions/serializers.py]: Hm, PyCharm takes out the second empty line when I "organize imports" -- I'll add it back in again

File: tests/test_fields.py (hunk 2 above)
Comment (ChrisBurch) [reply tests/test_fields.py]: Good eye! Fixing

**Comment by evenicoulddoit** [comment]
Sweet, thanks!

**Pull request merged**